lib/ast-to-react.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * @template T
+ * @typedef {import('react').ComponentType<T>} ComponentType<T>
+ */
```

Comment:
This needs to be in a separate comment block in order the template to not apply to all the types in the block.